doc/RELEASE_WALKTHROUGH.rst.txt

```diff
@@ -1,23 +1,31 @@
-This file contains a walkthrough of the NumPy 1.14.4 release on Linux.
+This file contains a walkthrough of the NumPy 1.14.5 release on Linux.
 The commands can be copied into the command line, but be sure to
-replace 1.14.4 by the correct version.
+replace 1.14.5 by the correct version.
 
 Release  Walkthrough
 ====================
 
+Backport Pull Requests, Write Release Note
+---------------------------------------------
+
+Changes that have been marked for this release must be backported to the
+maintenance/1.14.x branch. Summarize the changes in a release note
+doc/release/1.14.5-notes.rst
```

Comment:
In a new branch, the release notes will be present but need to be edited and reviewed. Usually the changelog can be generated at the same time once the backports are done.